app/views/devise/registrations/_personal_details.html.erb

```diff
@@ -29,6 +29,15 @@
   <% if org_admin %>
     <input type="hidden" id="original_org" value="<%= @user.org_id %>">
   <% end %>
+  
+    <% departments = (current_user.org.departments.present? ? current_user.org.departments : []) %>
```

Comment:
This should return [] when empty, so I don't think you need the conditional opperator